docs/installation.rst

```diff
@@ -482,72 +482,67 @@ connect to the databases you want to access through Superset.
-| SQLite           |                                       | ``sqlite://``                                   |
-+------------------+---------------------------------------+-------------------------------------------------+
-| SQL Server       | ``pip install pymssql``               | ``mssql://``                                    |
-+------------------+---------------------------------------+-------------------------------------------------+
-| Teradata         | ``pip install sqlalchemy-teradata``   | ``teradata://``                                 |
-+------------------+---------------------------------------+-------------------------------------------------+
-| Vertica          | ``pip install                         |  ``vertica+vertica_python://``                  |
-|                  | sqlalchemy-vertica-python``           |                                                 |
-+------------------+---------------------------------------+-------------------------------------------------+
-| Hana             | ``pip install hdbcli sqlalchemy-hana``|  ``hana://``                                    |
-|                  | or                                    |                                                 |
-|                  | ``pip install apache-superset[hana]`` |                                                 |
-+------------------+---------------------------------------+-------------------------------------------------+
++------------------+-------------------------------------------------------------------+-------------------------------------------------+
+| database         | pypi package                                                      | SQLAlchemy URI prefix                           |
++==================+===================================================================+=================================================+
+| Amazon Athena    | ``pip install "apache-superset[athena]"``                         | ``awsathena+jdbc://``                           |
++------------------+-------------------------------------------------------------------+-------------------------------------------------+
+| Amazon Redshift  | ``pip install  "apache-superset[redshift]"``                      | ``redshift+psycopg2://``                        |
++------------------+-------------------------------------------------------------------+-------------------------------------------------+
+| Apache Drill     | ``pip install "apache-superset[drill]"``                          | For the REST API:``                             |
+|                  |                                                                   | ``drill+sadrill://``                            |
+|                  |                                                                   | For JDBC                                        |
+|                  |                                                                   | ``drill+jdbc://``                               |
```

Comment:
As long as drill doesn't have an `extras_require` we probably need to state here that there is no PyPI package for drill yet, and just link to the GitHub repo.